bin/image-create-openstack.sh

```diff
@@ -106,7 +106,7 @@ echo "Verify md5 of present/uploaded image..."
 echo "List images available in OpenStack..."
 image_list="$(glance image-list)"
 image_id="$(printf '%s' "$image_list" |
-  grep "\s$KN_IMAGE_NAME\s" |
+  grep "\\s$KN_IMAGE_NAME\\s" |
```

Comment:
Just for the record :-) All the above changes are related to using a different version of shellcheck.

Reply:
Yes I was to quick asking for a review, Itt might be easier to see changes after provisioner-PR is merged

Reply:
Then I will merge against master again